README.md

```diff
@@ -33,13 +33,13 @@ $ npm install @placeholdersoft/typed-env
 
 ## Usage typed-env-cli
 
-### `diff-env`
+### `gen-diff-env`
```

Comment:
I think we can just call it `diff-env`, since output path is optional, the default behavior is just print out the result. how we will going to output the result became trivial in naming